libs/wingc/src/type_check/symbol_env.rs

```diff
@@ -114,7 +114,8 @@ pub enum LookupResult<'a> {
 	/// A matching symbol was found but it's not public
 	NotPublic(reference([a], [SymbolKind]), SymbolLookupInfo),
 	/// The symbol was not found in the environment, contains the name of the symbol or part of it that was not found
-	NotFound(Symbol),
+	/// If the lookup environment was a type environment, the type is also included
+	NotFound(Symbol, Option<TypeRef>),
```

Comment:
Maybe clarify in the comment with an example of a lookup in a type environment and a lookup not in a type environment

Reply:
added some wording, hope this helps